servlet/src/main/scala/org/http4s/servlet/Http4sServlet.scala

```diff
@@ -90,7 +91,17 @@ abstract class Http4sServlet[F[_]](service: HttpRoutes[F], servletIo: ServletIo[
               req.isSecure
             )),
           Request.Keys.ServerSoftware(serverSoftware),
-          ServletRequestKeys.HttpSession(Option(req.getSession(false)))
+          ServletRequestKeys.HttpSession(Option(req.getSession(false))),
+          Request.Keys.SecureSession(
+            (
+              Option(req.getAttribute("javax.servlet.request.ssl_session_id").asInstanceOf[String]),
+              Option(req.getAttribute("javax.servlet.request.cipher_suite").asInstanceOf[String]),
+              Option(req.getAttribute("javax.servlet.request.key_size").asInstanceOf[Int]),
+              Option(
+                req
```

Comment:
We are getting `javax.servlet.request.X509Certificate` and converting it to an array of `java.security.cert.X509Certificate`